src/ToggleButton.js

```diff
@@ -83,7 +89,7 @@ const ToggleButton = React.forwardRef(
           disabled && 'disabled',
         )}
         type={null}
-        active={!!checked}
+        active={!!checked || !!defaultChecked}
```

Comment:
this wouldn't work, the button would remain active when once checked changed, since `defaultChecked` is never updated

Reply:
A proper implementation i think would require controlling `checked` here and passing `defaultChecked` as the initial value

Reply:
@jquense Thank You for this feedback. I am going to work on fixing it.

Reply:
@jquense I have updated the PR kindly check it out. Thank You